src/lib.rs

```diff
@@ -22,4 +23,5 @@ pub mod prelude {
         AsKernelParam, CudaDevice, CudaDeviceBuilder, CudaSlice, DriverError, LaunchAsync,
         LaunchConfig, ValidAsZeroBits,
     };
+    pub use crate::cudnn::*;
```

Comment:
Let's not include this in the prelude for now

Reply:
ok